geoportailv3/static/js/search/searchdirective.js

```diff
@@ -0,0 +1,411 @@
+        var html = '<p>' + suggestion['translated_name'];
+        if (!goog.object.containsKey(suggestion, 'bglayer')) {
+          scope['click'] = goog.bind(function(event) {
+            this.showLayerinfo_(this.getLayerFunc_(suggestion));
+            event.stopPropagation();
+          }, this);
+          html += '<button ng-click="click($event)">i</button>';
+        } else {
+          html += ' (' + gettextCatalog.getString('Background') + ') ';
+        }
+        html += '</p>';
+        return $compile(html)(scope);
+      }, this)
+    })
+  }
+  ];
+
+  this['listeners'] = /** @type {ngeox.SearchDirectiveListeners} */ ({
+    selected: goog.bind(app.SearchDirectiveController.selected_, this)
+  });
+
+};
+
+
+/**
+ * @return {ol.FeatureOverlay} The feature overlay.
+ * @private
+ */
+app.SearchDirectiveController.prototype.createFeatureOverlay_ = function() {
+  var featureFill = new ol.style.Fill({
+    color: [255, 255, 0, 0.6]
+  });
+  var featureStroke = new ol.style.Stroke({
+    color: [255, 155, 55, 1],
+    width: 3
+  });
+  var featureOverlay = new ol.FeatureOverlay({
+    style: new ol.style.Style({
+      fill: featureFill,
+      stroke: featureStroke,
+      image: new ol.style.Circle({
+        radius: 10,
+        fill: featureFill,
+        stroke: featureStroke
+      })
+    })
+  });
+  featureOverlay.setMap(this['map']);
+  return featureOverlay;
+};
+
+
+/**
+ * @param {ngeo.CreateGeoJSONBloodhound} ngeoCreateGeoJSONBloodhound The ngeo
+ *     create GeoJSON Bloodhound service.
+ * @param {string} searchServiceUrl
+ * @return {Bloodhound} The bloodhound engine.
+ * @private
+ */
+app.SearchDirectiveController.prototype.createAndInitPOIBloodhound_ =
+    function(ngeoCreateGeoJSONBloodhound, searchServiceUrl) {
+  var url = searchServiceUrl + '?limit=5&query=%QUERY';
+  var bloodhound = ngeoCreateGeoJSONBloodhound(url, ol.proj.get('EPSG:3857'));
+  bloodhound.initialize();
+  return bloodhound;
+};
+
+
+/**
+ * @return {Bloodhound} The bloodhound engine.
+ * @private
+ */
+app.SearchDirectiveController.prototype.createAndInitLayerBloodhound_ =
+    function() {
+  var bloodhound = new Bloodhound({
+    datumTokenizer: Bloodhound.tokenizers.obj.whitespace('translated_name'),
+    queryTokenizer: Bloodhound.tokenizers.whitespace,
+    local: []
+  });
+  bloodhound.initialize();
+  return bloodhound;
+};
+
+
+/**
+ * @param {app.Themes} appThemes Themes Service
+ * @param {Bloodhound} bloodhound
+ * @param {angularGettext.Catalog} gettextCatalog Gettext catalog
+ * @private
+ */
+app.SearchDirectiveController.prototype.createLocalAllLayerData_ =
+    function(appThemes, bloodhound, gettextCatalog) {
+  bloodhound.clear();
+  this.layers_ = [];
+  appThemes.getBgLayers().then(
+      goog.bind(function(bgLayers) {
+        goog.array.forEach(bgLayers, function(bgLayer) {
+          var bgLayerObj = {};
+          bgLayerObj.bglayer = bgLayer;
+          bgLayerObj.translated_name =
```

Comment:
Or maybe you want to use `['translated_name']` instead to be consistent with what's used elsewhere in your code? That might explain the problems you're still having in non-debug mode?